pkg/cmd/util/clientcmd/factory_client_access.go

```diff
@@ -51,7 +63,22 @@ type ring0Factory struct {
 type ClientAccessFactory interface {
 	kcmdutil.ClientAccessFactory
 
+	// TODO: this should be removed when we finally get rid of pkg/client
```

Comment:
we still need core kube client and probably some other clients as well (deployments?) i would actually suggest, while you are here, to remove remove this function and create:

```
KubeInternalClient(name string) (kclientsetinternal.Interface, error)
DeprecatedOpenshiftClient(name string) (osclient.Interface, error)
```

Then fix the CLI so at least we can grep for the usage of Deprecated client and replace step-by-step :)

Reply:
> we still need core kube client and probably some other clients as well (deployments?) i would actually suggest, while you are here, to remove remove this function and create:

The kube client shoudl already be accessible on the stock factory.